core/server/master/src/main/java/alluxio/master/file/InodeSyncStream.java

```diff
@@ -751,7 +757,8 @@ private void loadMetadata(LockedInodePath inodePath, LoadMetadataContext context
             }
             LoadMetadataContext loadMetadataContext =
                 LoadMetadataContext.mergeFrom(LoadMetadataPOptions.newBuilder()
-                    .setLoadDescendantType(LoadDescendantPType.NONE)
+                    .setLoadDescendantType(type.equals(LoadDescendantPType.ONE)
```

Comment:
we should set loadDescendantType to all if it was all to begin with.. setting it to NONE disables recursive loading of the paths.

Reply:
If `loadDescendantType` is `ALL` won't this trigger recursive calls to this function on L769?
And if we trigger recursive calls, then won't be also be losing the parallelism of the inode metadata loading? It seems like if it's set to `ALL` this function now loads everything in a single-threaded manner?

Reply:
Please add a comment here, so it is easier to understand.

Reply:
LoadMetadataForPath does a maximum of one level and a minimum of 0 level now.